packages/kit/test/apps/basics/src/routes/errors/_tests.js

```diff
@@ -211,4 +211,17 @@ export default function (test, is_dev) {
 			assert.ok(!has_stack_trace, 'Stack trace is visible');
 		}
 	});
+
+	test(
+		'server-side 4xx status without error from load()',
```

Comment:
this should probably have a client-side test as well. you can copy the tests from https://github.com/sveltejs/kit/pull/1355

Reply:
will do